stories/voice-search.stories.js

```diff
@@ -99,7 +99,7 @@ storiesOf('VoiceSearch', module)
               if (isSpeechFinal) {
                 return `Final Transcript: ${transcript}`;
               } else {
-                return `Interim Transcript: ${transcript}`;
```

Comment:
Prefer squashing this in the first commit introducing the error message (making the history cleaner)